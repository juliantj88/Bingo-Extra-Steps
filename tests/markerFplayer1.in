1
test1
test2
1 R 5
2 U 1
5 R 5
4 U 5
0 B 2
0 U 2
3 B B
0 Y B
0 L B
save markerFplayer1
4