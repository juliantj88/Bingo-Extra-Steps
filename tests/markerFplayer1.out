test1
test2
-4
-7
1
F...............
RRRR
RLLB
BBUU
RBYY
RBYU
byrul
lByru
ulbyr
rulby
yRulb
byrUl
lbyru
ulbyr
rulby
yrulb
.
..
...
....
.....
.
U.
...
....
UUU..
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYU
0